.travis.yml

```diff
@@ -32,6 +32,9 @@ services:
 
 sudo: false
 
+before_install:
+  - composer config -g github-oauth.github.com $CI_USER_TOKEN
```

Comment:
I hope I'm wrong but I think all this features are only available for private repos. I had similar doubt about having private tokens in public repos in travisCI http://stackoverflow.com/questions/31209313/in-travis-public-repository-how-to-add-a-secure-variable-that-works-on-pull-requ but no luck.